README.md

```diff
@@ -8,32 +8,24 @@
 
 A tool for debugging and assessing floating point precision and reproducibility.
 
-   * [Using Verificarlo through its Docker image](#using-verificarlo-through-its-docker-image)
    * [Installation](#installation)
-      * [Example on x86_64 Ubuntu 20.04 release with Fortran support](#example-on-x86_64-ubuntu-2004-release-with-fortran-support)
-      * [Checking installation](#checking-installation)
+   * [Using Verificarlo through its Docker image](#using-verificarlo-through-its-docker-image)
    * [Usage](#usage)
+   * [Branch instrumentation](#branch-instrumentation)
    * [Examples and Tutorial](#examples-and-tutorial)
    * [Backends](#backends)
-      * [IEEE Backend (libinterflop_ieee.so)](#ieee-backend-libinterflop_ieeeso)
-      * [MCA Backend (libinterflop_mca.so)](#mca-backend-libinterflop_mcaso)
-      * [MCA-MPFR Backend (libinterflop_mca_mpfr.so)](#mca-mpfr-backend-libinterflop_mca_mpfrso)
-      * [Bitmask Backend (libinterflop_bitmask.so)](#bitmask-backend-libinterflop_bitmaskso)
-      * [Cancellation Backend (libinterflop_cancellation.so)](#cancellation-backend-libinterflop_cancellationso)
-      * [VPREC Backend (libinterflop_vprec.so)](#vprec-backend-libinterflop_vprecso)
-   * [Verificarlo inclusion / exclusion options](#verificarlo-inclusion--exclusion-options)
-   * [Function instrumentation](#function-instrumentation)
-   * [VPREC custom precision](#vprec-custom-precision)
+   * [Delta Debug and inclusion / exclusion options](#delta-debug-and-inclusion--exclusion-options)
+   * [VPREC Function instrumentation](#vprec-function-instrumentation)
    * [Postprocessing](#postprocessing)
```

Comment:
This should be good since ad8c9fc554b719b223a8297a05157349fda2b5a7, the documentation has been put at the end of `ad8c9fc554b719b223a8297a05157349fda2b5a7` and linked from the README.